dogfooding/lib/screens/lobby_screen.dart

```diff
@@ -14,17 +19,138 @@ class LobbyScreen extends StatelessWidget {
   final ValueChanged<CallConnectOptions> onJoinCallPressed;
   final Call call;
 
+  @override
+  State<LobbyScreen> createState() => _LobbyScreenState();
+}
+
+class _LobbyScreenState extends State<LobbyScreen> {
+  RtcLocalAudioTrack? _microphoneTrack;
+  RtcLocalCameraTrack? _cameraTrack;
```

Comment:
I don't understand why we need to keep track of these variables in this Widget. Can you please elaborate a bit?

Reply:
Those are created and destroyed by `StreamLobbyVideo` when turning camera/microphone on/off and have to be stored so that later when call is created they are passed in CallConnectoptions to initialize call screen

Reply:
OK I got it now, thanks!